### ISSUE TITLE
jpeg sizing issue on export

### ISSUE THREAD

**Issue by ChefAndy** [harvard-lil/h2o#1498]
Some jpegs aren't sized correctly upon export. 

I noticed in one of my testing casebooks after placing a some new test images from the LIL site that some were rendered with the wrong aspect ratio. I initially assumed it was layout interference, but realized that the pictures the working pictures came out of a DSLR but the janky ones were shot with phones. The images themselves seem valid— embedding them into the same docx in Word worked fine. Upon saving, the new embed didn't create a new file in the doc's media directory, so it was definitely using the same file, but the old embeds still looked wrong.

Sure enough, the dimensions were hard-coded in document.xml:
```
<a:xfrm>
    <a:off x="0" y="0"/>
    <a:ext cx="3810000" cy="2540000"/>
</a:xfrm>
```
The aspect ratio is way off. the native word embed uses these values:
```
<a:xfrm>
    <a:off x="0" y="0"/>
    <a:ext cx="4114800" cy="4114800"/>
<a:xfrm>
```
Manually matching up the X and Y values for the opencasebook.org embed fixed the problem.

**Comment by ChefAndy** [comment]
Ah— It looks like Pandoc just fixed a similar bug for PNGs in 2.14.1:
https://github.com/jgm/pandoc/issues/7405

A google groups discussion:
https://groups.google.com/g/pandoc-discuss/c/cucyZhfe45o/m/ZRaUXO1bAgAJ

> If I look at the document.xml in the unzipped version of the pandoc-generated docx, I see this, which provides a 1.5 ratio of cx to cy:
>```
><a:xfrm>
>    <a:off x="0" y="0"/>
>    <a:ext cx="3810000" cy="2540000"/>
></a:xfrm>
>```
> If I use image magic's format command, the original tiff has the following relevant (I think) properties:
> -  Geometry: 1758x798+0+0
> -  Resolution: 300x300
> -  Print size: 5.86x2.66
> -  Units: PixelsPerInch

Since were seeing this issue in 2.14.2, I reckon we've just run up against a unusual instance of that. I'll dig in a little more tomorrow and file an issue if needed.

**Comment by lizadaly** [comment]
Since there's no reproducible case here, I'm closing this for now, but this will be helpful debugging info if we see an instance in the wild.
